pkg/proxy/ipvs/proxier.go

```diff
@@ -1384,16 +1387,24 @@ func (proxier *Proxier) syncEndpoint(svcPortName proxy.ServicePortName, onlyNode
 	return nil
 }
 
-func (proxier *Proxier) cleanLegacyService(atciveServices map[string]bool, currentServices map[string]*utilipvs.VirtualServer) {
+func (proxier *Proxier) cleanLegacyService(activeServices map[string]bool, currentServices map[string]*utilipvs.VirtualServer) {
 	unbindIPAddr := sets.NewString()
-	for cS := range currentServices {
-		if !atciveServices[cS] {
-			svc := currentServices[cS]
-			err := proxier.ipvs.DeleteVirtualServer(svc)
+	for cs := range currentServices {
+		svc := currentServices[cs]
+		for _, excludedCidr := range proxier.excludeCIDRs {
+			_, net, err := net.ParseCIDR(excludedCidr)
```

Comment:
bad naming of variable ... conflicts with imported package name

Reply:
Done.